lib/main.coffee

```diff
@@ -8,6 +8,9 @@ module.exports =
       scopes:
         '.source.jade':
           default: false
+    ignoreMarkdownLineBreak:
+      type: 'boolean'
```

Comment:
Changed the name to `keepMarkdownLineBreakWhitespace` and added the following description:

> Markdown uses two ore more spaces at the end of a line to signify a line break. Enable this option to keep this whitespace, even if other settings would remove it.